src/dispatch/incident/flows.py

```diff
@@ -265,6 +271,30 @@ def incident_create_flow(*, organization_slug: str, incident_id: int, db_session
             db_session=db_session,
         )
 
+    return incident
+
+
+@background_task
+def incident_create_resources_flow(
+    *, organization_slug: str, incident_id: int, db_session=None
+) -> Incident:
+    """Creates all resources required for new incidents and initiates incident response workflow."""
+    # we get the incident
+    incident = incident_service.get(db_session=db_session, incident_id=incident_id)
+
+    # we create the incident resources
+    return incident_create_resources(incident=incident, db_session=db_session)
+
+
+@background_task
+def incident_create_flow(*, organization_slug: str, incident_id: int, db_session=None) -> Incident:
+    """Creates all resources required for new incidents and initiates incident response workflow."""
+    # we get the incident
+    incident = incident_service.get(db_session=db_session, incident_id=incident_id)
+
+    # we create the incident resources
+    incident_create_resources(incident=incident, db_session=db_session)
+
     event_service.log_incident_event(
```

Comment:
I wonder if this should live inside the new `incident_create_resources` function now.

Reply:
Perhaps the `log_incident_event` for "Incident notifications sent" should live in `send_incident_created_notifications`

Reply:
Done.